python/paddle/fluid/optimizer.py

```diff
@@ -839,8 +839,10 @@ class SGDOptimizer(Optimizer):
         parameter_list (list, optional):  List of ``Variable`` names to update to minimize ``loss``. \
             This parameter is required in dygraph mode. \
             The default value is None in static mode, at this time all parameters will be updated.
-        regularization: A Regularizer, such as :ref:`api_fluid_regularizer_L2DecayRegularizer`. \
-            Optional, default is None.
+        regularization (WeightDecayRegularizer, optional): The strategy of regularization. There are two method:
```

Comment:
You should explain 2 things:

- If the parameter has set regularizer using `ParamAttr`, the regularization settings in optimizer for this parameter is ignored.
- If the parameter does not set  regularizer using `ParamAttr`, the regularization of this parameter would be the one in optimizer settings.

Reply:
Done.